docker/aligned_base.Dockerfile

```diff
@@ -31,12 +28,83 @@ RUN go install github.com/maoueh/zap-pretty@latest
 RUN go install github.com/ethereum/go-ethereum/cmd/abigen@latest
 RUN go install github.com/Layr-Labs/eigenlayer-cli/cmd/eigenlayer@latest
 
+# Install rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
 WORKDIR /aligned_layer
 
 COPY Makefile .
-COPY operator ./operator
-COPY batcher/aligned-sdk ./batcher/aligned-sdk
+#COPY operator ./operator
+#COPY batcher/aligned-sdk ./batcher/aligned-sdk
```

Comment:
why is this commented?

Reply:
Same as above. I was testing stuff and now it is no longer needed to be commented.